projects/js-packages/publicize-components/src/components/post-publish-share-status/share-status.tsx

```diff
@@ -52,7 +52,14 @@ export function ShareStatus() {
 		);
 	}
 
+	if ( ! shareStatus.done ) {
+		// If we are here, it means that polling has finished/timedout
+		// but we don't know the share status yet.
+		return <span>{ __( 'The request to share your post is still in progress.', 'jetpack' ) }</span>;
```

Comment:
Should we leave some guidance here, like "come back later" or "refresh in a few minutes"?

Reply:
I was also thinking about the same. Thanks. Adding that to the sentence.

Reply:
Done in 5b84870e7c18498fbf8574624cb603ecdf852e39

<img width="290" alt="image" src="https://github.com/user-attachments/assets/387bb0d4-42ee-4bff-8a7f-9916f4aa5fa9">